src/problem/MFEMProblem.C

```diff
@@ -23,6 +24,9 @@ MFEMProblem::MFEMProblem(const InputParameters & params)
     _outputs(),
     _exec_params()
 {
+  // logger.set_level(spdlog::level::info);
```

Comment:
Logger line here should be removed. In MOOSE we can take advantage of the existing informational messaging system [(docs)](https://mooseframework.inl.gov/framework_development/error_warning_messaging.html) and use `mooseInfo(<args>)` for info logs.

Reply:
Will do. There are also other commented-out spdlog lines scattered around the code. I'll take the opportunity to remove them too, if that's alright